targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_BLE/device/system_psoc6_cm4.c

```diff
@@ -277,13 +281,22 @@ void SystemInit(void)
 *******************************************************************************/
 void mbed_sdk_init(void)
 {
+#if !defined(COMPONENT_SPM_MAILBOX)
```

Comment:
please remove this code until you actually add the BLE kit as a PSA target

Reply:
See https://github.com/ARMmbed/mbed-os/pull/10308#discussion_r275443336